### DISCUSSION TITLE
Deno不只是个Javascript运行时 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#194] [General]
# Deno不只是个Javascript运行时 - 愧怍的小站

Deno 是一个安全的 JavaScript 和 TypeScript 运行时，作者是 Ryan Dahl（也是 Node.js 的原作者）。Deno 的诞生之初是为了解决 2009 年首次设计 Node.js 时的一些疏忽。我认为这种改造动机很有道理，因为我相信每个程序员都希望有机会能重写他们已有 10 年历史的代码。

https://kuizuo.cn/deno-is-not-only-a-javascript-runtime#comments

**Comment by zhzcor** [comment]
ffff

**Reply by zhzcor** [reply]
dd
